src/lib/components/product.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+  import { fade } from 'svelte/transition'
+
+  type Vermouth = {
+    brand: string
+    name: string
+    image: string
+  }
+  export let vermouths: Vermouth[]
+</script>
+
+<div class="container">
+  {#each vermouths as { image, name, brand }, index}
+    <!-- <a href="#"> -->
+    <div in:fade={{ duration: 300, delay: index * 200 }} class="product">
```

Comment:
only this part should be the Product component - we don't want each and container and all that in a single Product component. 

And don't change the component name, best practice to have product.svelte, each loop in parent